packages/modules/src/modules/header/EoYUSMomentHeader.stories.tsx

```diff
@@ -5,6 +5,12 @@ import { Header } from './EoYUSMomentHeader';
 import { css } from '@emotion/core';
 import { brand } from '@guardian/src-foundations';
 
+const givingTuesdayStart = new Date('2021-11-29T17:00:00'); //remove "Subscribe" Monday 12:00 PM EST
+const givingTuesdayEnd = new Date('2021-11-01T09:00:00'); //re-add "Subscribe" on Wednesday morning GMT
```

Comment:
givingTuesdayEnd is first of December: `const givingTuesdayEnd = new Date('2021-12-01T09:00:00');`

Reply:
Hi @GHaberis, as the date library is indexed from 0 we had to reduce the month by 1.